dmcontent/govuk_frontend.py

```diff
@@ -64,10 +64,12 @@
 # set this in your app to change the behaviour of this code.
 govuk_frontend_version = (2, 13, 0)
 
+MultiQuestionToRender = List[Union[dict, Markup]]
```

Comment:
Rather than making a type that is specific to multiquestions, I would make a type `Renderable = Union[dict, str, Markup, List[Union[dict, str, Markup]]`, as that describes everything that can be produced by `from_question` or handled by `render`

Reply:
Good idea - added in f24d2ae!